assets/js/components/data/index.js

```diff
@@ -199,7 +199,7 @@ const dataAPI = {
 		} ).then( ( results ) => {
 			each( results, ( result, key ) => {
 				if ( ! keyIndexesMap[ key ] ) {
-					console.error( 'data_error', 'unknown response key ' + key ); // eslint-disable-line no-console
+					console.debug( 'data_error', 'unknown response key ' + key ); // eslint-disable-line no-console
```

Comment:
I've downgraded the severity of the logging here as this is only really relevant for debugging and not something which is really likely to ever happen under normal conditions.

It also happens regularly in E2E for tests where we return a mocked response for a batch request since only uncached request keys will be requested on subsequent calls.